inc/util.h

```diff
@@ -78,19 +83,6 @@ int64_t parseIso8601Time(const char *str);
 
 uint64_t parseUnsignedInt(const char *str);
 
-// base64 encode bytes.  The output buffer must have at least
-// ((4 * (inLen + 1)) / 3) bytes in it.  Returns the number of bytes written
-// to [out].
-int base64Encode(const unsigned char *in, int inLen, char *out);
```

Comment:
I removed them from `util.c` in the same commit (584bf483e778a469777c2fbf72fa89a8e05b658d).